warehouse/scripts/run_and_upload.py

```diff
@@ -284,12 +291,59 @@ def get_command(*args) -> List[str]:
                     "--dbt_docs_url",
                     "https://dbt-docs.calitp.org",
                     "--metabase_database",
-                    "Data Marts (formerly Warehouse Views)",
+                    (
+                        "Data Marts (formerly Warehouse Views)"
+                        if target.startswith("prod")
+                        else "(Internal) Staging Warehouse Views"
+                    ),
                     "--dbt_schema_excludes",
                     "staging",
                     "payments",
-                ]
+                ],
+                env={
+                    **os.environ,
+                    "COLUMNS": "300",  # we have to make this wide enough to avoid splitting log lines
+                },
+                capture_output=True,
             )
+
+            with open("./target/manifest.json") as f:
+                manifest = Manifest(**json.load(f))
+            matches = {}
+            for line in p.stdout.decode().splitlines():
+                print(line)
+                for pattern in (
+                    r"WARNING\s+Model\s(?P<dbt_schema>\w+)\.(?P<model>\w+)\snot\sfound\sin\s(?P<db_schema>\w+)",
```

Comment:
Just noting we'll have to be sure to come back to this later if we change the strings underlying this pattern we're looking for - since we don't have a direct linkage here to trace back to the strings themselves in code, just a regex, it's potentially brittle/easily missed in future changes.